.github/actions/gradle/experiment-1/action.yml

```
@@ -108,7 +108,13 @@ runs:
         cat $RECEIPT_FILE >> $GITHUB_STEP_SUMMARY
       shell: bash
     - name: Archive receipt
+      id: upload-artifact
       uses: actions/upload-artifact@v4
       with:
-        name: experiment-1-receipt
+        name: experiment-1-receipt-${{ github.job }}${{ strategy.job-total > 1 && format('-{0}', strategy.job-index) || '' }}
         path: gradle-enterprise-gradle-build-validation/.data/01-validate-incremental-building/latest*/exp1-*.receipt
+    - name: Add artifact link to summary
+      run: |
+        echo "-------------" >> $GITHUB_STEP_SUMMARY
+        echo "Download receipt: [${{ steps.upload-artifact.outputs.artifact-url }}](${{ steps.upload-artifact.outputs.artifact-url }})" >> $GITHUB_STEP_SUMMARY
```

It's already a link. Is it necessary to wrap it like this?

```suggestion
        echo "Download receipt: ${{ steps.upload-artifact.outputs.artifact-url }}" >> $GITHUB_STEP_SUMMARY
```

---

Good point 😅
I adjusted the PR